src/app/campaign-details/campaign-details.component.scss

```diff
@@ -150,16 +128,20 @@ h1 {
 }
 
 #campaign-highlights-beneficiaries-and-categories-mobile {
+  display: block;
+  margin-top: 30px;
+}
+
+#campaign-highlights-beneficiaries-and-categories-desktop {
   display: none;
 }
 
-@media screen and (max-width: 960px){
-  #campaign-highlights-beneficiaries-and-categories-desktop {
+@media #{$breakpoint-lg} {
```

Comment:
Just tweaking this so effective breakpoints are consistent. I'm not really convinced we need 2 totally separate elements for different screen sizes here, but don't have time to refactor that out today.